Multy/Cells/Create Wallet Name/CreateWalletNameTableViewCell.swift

```diff
@@ -0,0 +1,20 @@
+//Copyright 2017 Idealnaya rabota LLC
+//Licensed under Multy.io license.
+//See LICENSE for details
+
+import UIKit
+
+class CreateWalletNameTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var walletNameTF: UITextField!
+    
+
+    override func awakeFromNib() {
```

Comment:
Here and other places too, does that actually makes any sense to overload a method from base (super) class to just invoke same method from super class?

Reply:
This lines created with the class and helps to understand where put additional functionality if needed (it similar to viewDidLoad, viewWillAppear and etc.).